v2/gcs-to-sourcedb/src/test/resources/GCSToSourceDbInterleaveMultiShardIT/spanner-schema.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE parent1 (
+  id INT64 NOT NULL,
+  update_ts TIMESTAMP,
+  in_ts TIMESTAMP,
+  migration_shard_id STRING(50),
+) PRIMARY KEY(id);
+
+CREATE TABLE parent2 (
+  id INT64 NOT NULL,
+  update_ts TIMESTAMP,
+  in_ts TIMESTAMP,
+  migration_shard_id STRING(50),
+) PRIMARY KEY(id);
+
+CREATE TABLE child11 (
```

Comment:
child11 has foreign key in MySQL schema. Is this intended? Or should we have either interleaving or FK here in spanner as well?

Reply:
It was intended. Actually I took the same schema that SMT generated, looks like Spanner FK were missed. Spanner FK will in schema will guarantee ordered writes : which am doing programmatically, so keeping it such.